Remove support for `HttpResponseMessage` assertions (#2909)
fluentassertions/fluentassertions: docs/_pages/httpresponsemessages.md

nav: "sidebar"
---

> The support for assertions on this type was removed in FluentAssertions 8.0. If you need to do assertions on this type please refer to [FluentAssertions.Web](https://github.com/adrianiftode/FluentAssertions.Web) in future.

Reviewer: ```suggestion
The support for assertions on this type was removed in FluentAssertions 8.0. If you need to do assertions like these, we recommend you adopt [FluentAssertions.Web](https://github.com/adrianiftode/FluentAssertions.Web).
```
